src/control/server/storage/bdev/backend_json_test.go

```diff
@@ -74,7 +74,7 @@ func TestBackend_newSpdkConfig(t *testing.T) {
 		"config validation failure": {
 			class:          storage.ClassNvme,
 			devList:        []string{"not a pci address"},
-			expValidateErr: errors.New("unexpected pci address"),
+			expValidateErr: errors.New("valid PCI addresses"),
```

Comment:
As suggested by @kjacque ,for a newcomer such as me, the old error message is more relevant.